tools/build_chain.sh

```diff
@@ -356,16 +373,8 @@ gen_chain_cert_gm() {
     chaindir=$path
     mkdir -p $chaindir
 
-    generate_gmsm2_param "gmsm2.param"
-	$TASSL_CMD genpkey -paramfile gmsm2.param -out $chaindir/gmca.key
-	$TASSL_CMD req -config gmcert.cnf -x509 -days 3650 -subj "/CN=${name}/O=fisco-bcos/OU=chain" -key $chaindir/gmca.key -extensions v3_ca -out $chaindir/gmca.crt
-
-    if $(cp gmcert.cnf gmsm2.param $chaindir)
-    then
-        echo "build chain ca succussful!"
-    else
-        echo "please input at least Common Name!"
-    fi
+	$TASSL_CMD genpkey -paramfile gmsm2.param -out "$chaindir/gmca.key" 2> /dev/null
+	$TASSL_CMD req -config gmcert.cnf -x509 -days 3650 -subj "/CN=${name}/O=fisco-bcos/OU=chain" -key $chaindir/gmca.key -extensions v3_ca -out $chaindir/gmca.crt 2> /dev/null
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/FISCO-BCOS/FISCO-BCOS/pullRequest?prid=5331982)